django/tasks/register.py

```diff
@@ -0,0 +1,28 @@
+class AlreadyRegistered(Exception):
+    pass
+
+
+class TaskDoesNotExist(Exception):
+    pass
+
+
+class TaskRegistry(object):
+    def __init__(self):
+        self._registry = {}
+
+    def register(self, task):
+        if task.name in self._registry:
```

Comment:
Yes it does, especially since we just added fallback to default queue by default it must remain so.